File: aulas/2026/aula-00-introducao/exercicios/01-zerinho-ou-um/solucao.py
import sys


for linha in sys.stdin:
  alice, beto, clara = map(int, linha.split())

  if alice != beto and beto == clara:
    print("A")
  elif beto != alice and alice == clara:
    print("B")
  elif clara != alice and alice == beto:
    print("C")
  else:
    print("*")

File: aulas/2026/aula-00-introducao/exercicios/01-zerinho-ou-um/solucao.cpp
#include <iostream>

using namespace std;

int main()
{
  int alice;
  int beto;
  int clara;

  while (cin >> alice >> beto >> clara)
  {
    if (alice != beto && beto == clara)
    {
      cout << "A" << endl;
    }
    else if (beto != alice && alice == clara)
    {
      cout << "B" << endl;
    }
    else if (clara != alice && alice == beto)
    {
      cout << "C" << endl;
    }
    else
    {
      cout << "*" << endl;
    }
  }

  return 0;
}
